packages/react/src/useUploadThing.ts

```diff
@@ -46,7 +54,10 @@ export const INTERNAL_uploadthingHookGen = <TRouter extends FileRouter>() => {
     const uploadProgress = useRef(0);
     const fileProgress = useRef<Map<string, number>>(new Map());
 
-    const permittedFileInfo = useEndpointMetadata(endpoint as string);
+    const permittedFileInfo = useEndpointMetadata(
+      initOpts.url,
+      endpoint as string,
```

Comment:
not really part of this, but is it possible for useEndpointMetadata to accept a `TEndpoint` directly, instead of needing to typecast?